src/components/core/InteractiveLink/NavigateConfirmation.tsx

```diff
@@ -0,0 +1,46 @@
+import { useModalActions } from 'contexts/modal/ModalContext';
+import styled from 'styled-components';
+import { Button } from 'components/core/Button/Button';
+import colors from 'components/core/colors';
+import Spacer from 'components/core/Spacer/Spacer';
+
+export default function VerifyNavigationPopover({ href }: { href: string }) {
+  const { hideModal } = useModalActions();
+
+  return (
+    <StyledConfirmation>
+      <Spacer height={16} />
+      <ButtonContainer>
+        <StyledCancelButton onClick={hideModal}>Cancel</StyledCancelButton>
+        <StyledButton
+          onClick={() => {
+            window.open(href);
+            hideModal();
+          }}
+        >
+          Navigate
+        </StyledButton>
```

Comment:
could we add a bit more left/right padding on the button? example of another one below

<img width="96" alt="image" src="https://user-images.githubusercontent.com/12162433/182529879-3d24c023-8744-4a97-9d79-14e0418c5f17.png">

<img width="152" alt="image" src="https://user-images.githubusercontent.com/12162433/182529830-e2ec3de3-ffd6-4edd-8138-4b5c47e59ec5.png">